docs/embedding_endpoints.md

```diff
@@ -50,24 +60,25 @@ MemGPT supports running embeddings with any Hugging Face model using the [Text E
 ...
 ```
 
-## Local Embeddings 
+## Local Embeddings
 
-MemGPT can compute embeddings locally using a lightweight embedding model [`BAAI/bge-small-en-v1.5`](https://huggingface.co/BAAI/bge-small-en-v1.5). 
+MemGPT can compute embeddings locally using a lightweight embedding model [`BAAI/bge-small-en-v1.5`](https://huggingface.co/BAAI/bge-small-en-v1.5).
 
 > 🚧 Local LLM Performance
-> 
-> The `BAAI/bge-small-en-v1.5` was chosen to be lightweight, so you may notice degraded performance with embedding-based retrieval when using this option. 
+>
+> The `BAAI/bge-small-en-v1.5` was chosen to be lightweight, so you may notice degraded performance with embedding-based retrieval when using this option.
 
-To compute embeddings locally, install dependencies with: 
-```
+To compute embeddings locally, install dependencies with:
+
+```sh
 pip install `pymemgpt[local]`
 ```
-Then, select the `local` option during configuration: 
-```
-> memgpt configure                                                                                    
+
+Then, select the `local` option during configuration:
+
+```sh
```

Comment:
eg should this be `text` too? or an `sh` block going into a `text` block? to avoid weird highlighting on the CLI outputs